### PR TITLE
IBX-6383: Remove Allure reports integration from CI

### PR DESCRIPTION
**Pull request by micszo**
**JIRA**: [IBX-6383](https://issues.ibexa.co/browse/IBX-6383)

The goal of the task is to remove the whole Allure reports integration from our test code in order to shut the server hosting it down.

Complementary to: https://github.com/ezsystems/BehatBundle/pull/293

### CODE REVIEW

File: .github/workflows/browser-tests.yml [hunk 1]
```diff
@@ -275,7 +265,6 @@ jobs:
                  {\"blocks\": [{\"type\": \"section\",\"text\": {\"type\": \"mrkdwn\",\"text\": \"
                  $RESULT_EMOJI *$GITHUB_REPOSITORY*:*$GITHUB_REF_NAME* ($GITHUB_ACTOR) |
                  <$GITHUB_SERVER_URL/$GITHUB_REPOSITORY/actions/runs/$GITHUB_RUN_ID|Details> |
```
Comment (mnocon) [inline .github/workflows/browser-tests.yml:267]: Our current notifications look like this:
```
ibexa/product-catalog:main (Steveb-p) | Details | Report (1/1)
```

If I understand correctly the notification will look like this:
```
ibexa/product-catalog:main (Steveb-p) | Details | (1/1)
```

Just a suggestion: what about something like this?
```
ibexa/product-catalog:main (Steveb-p) | Details | 1/1
```

**Review by mnocon — APPROVED** [review]
Thanks!

**Review by bogusez — APPROVED** [review]

File: .github/workflows/browser-tests.yml (hunk 1 above)
Comment (tomaszszopinski) [reply .github/workflows/browser-tests.yml:267]: Last one should be ok 👍

**Review by tomaszszopinski — APPROVED** [review]

File: .github/workflows/browser-tests.yml (hunk 1 above)
Comment (micszo) [reply .github/workflows/browser-tests.yml:267]: Done in https://github.com/ibexa/gh-workflows/pull/42/commits/3e225bb35788f4e5062988e98ff67a36aac9ebb6.

**Pull request merged**